index.html

```diff
@@ -135,6 +137,9 @@ <h3>Latest news about <span>[populate name/industry/sector from the input form]<
         </section>
     </main>
 
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
I actually already included this link to jQuery on line 142/147 below, so you can just remove this line here.